Marksman/Server.fs

```diff
@@ -715,4 +721,57 @@ type MarksmanServer(client: MarksmanClient) =
 
         AsyncLspResult.success tokens
 
+    override this.TextDocumentCodeAction(opts: CodeActionParams) =
+        let state = requireState ()
+        let docPath = opts.TextDocument.Uri |> PathUri.fromString
+
+        let documentBeginning =
+            { Start = { Line = 0; Character = 0 }
```

Comment:
nit: `open Marksman.Misc` (if not already) to load `Position` and `Range` extensions and then `Position.Mk(0, 0)` or `Range.Mk(0, 0, 0, 0)`.